kdumpctl

```diff
@@ -1794,7 +1907,7 @@ main()
 		;;
 	test)
 		shift
-		kdump_test "$@"
+		kdump_test
```

Comment:
maybe missed $@ here?

Reply:
Yes, thanks for pointing it out, it should be "$@". I will fix it in v2.

Reply:
See v2 which fixed the bug.